documentation/3.4.0/setup/native/project-with-survey/mac-os/README.md

```
@@ -0,0 +1,354 @@
+        cd user/src && npx sequelize-cli db:seed:all && cd ../../
+        ```
+
+8.  **Enabling Citus And Setting Distribution Columns (Optional)**
+
+    To boost performance and scalability, users can opt to enable the Citus extension. This transforms PostgreSQL into a distributed database, spreading data across multiple nodes to handle large datasets more efficiently as demand grows.
+
+    > NOTE: Currently only available for Linux based operation systems.
+
+    1. Download user `distributionColumns.sql` file.
+
+        ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)